helm-charts/code-annotation/templates/deployment.yaml

```diff
@@ -50,7 +50,7 @@ spec:
                   name: {{ $secretName  }}
                   key: jwt_signing_key
             - name: CAT_GA_TRACKING_ID
-              value: "{{ .Values.deployment.gaTrackingID }}"
+              value: "{{ required "gaTrackingID is required" .Values.deployment.gaTrackingID }}"
```

Comment:
I found two different ways to write it:

wrapping the value [with quotes](https://github.com/src-d/code-annotation/pull/218/files#diff-f03305ba93fd99a4f9a9c72bcb7d5d06R26):
```
image: "{{ .Values.image.repository }}:{{ required "Image tag is required" .Values.image.tag }}"
```

leaving the value [without wrapper quotes](https://github.com/dpordomingo/code-annotation/blob/93a4e4259bd4d9c1684a47af0cc1a2bc1cf70c5d/helm-charts/code-annotation/templates/ingress.yaml#L20):
```
- host: {{ required "Hostname is missing" .Values.ingress.hostname }}
```

@rporres should I use one style for this `CAT_GA_TRACKING_ID` env value?

Reply:
All env variables MUST be strings, so they must be quoted to avoid YAML interpreting as numbers